idea/static/idea/css/idea.css

```diff
@@ -389,7 +387,7 @@ ul.section-nav li.active a {
 #date-range {
     margin-top: 20px;
 }
-.sidebar .challenge-banner #challenge-link {
+.challenge-banner #challenge-link {
```

Comment:
I scoped this as `.idea-hero .challenge-banner #challenge-link` since I wasn't sure where else `.challenge-banner` appeared, but I'm not sure if that's necessary or not.